File: seccion_6/ej.py
class procesador:
    def instalar(self):
        print("procesador instalado")


class computador:
    def __init__(self):
        self.procesador = procesador()

    def instalar(self):
        self.procesador.instalar()
        print("Computador listo para usar")


c = computador()
c.instalar()

File: seccion_6/ej.java
class procesador {
 void instalar() {
 System.out.println("procesador instalado");
 }
}
class computador {
 procesador procesador = new procesador();
 void instalar() {
 procesador.instalar();
 System.out.println("Computador listo para usar");
 }
}
public class ej {
 public static void main(String[] args) {
 computador c = new computador();
 c.instalar();
 }
}